action.yml

```diff
@@ -149,51 +150,53 @@ runs:
           fi
         }
 
-        if [[ ${{ inputs.gitopsenabled }} == "true" ]]; then
-          git config --global user.email "${{ inputs.gitopsemail }}" && git config --global user.name "${{ inputs.gitopsuser }}"
+        # download and install yq
+        sudo wget -O /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+        sudo chmod +x /usr/local/bin/yq
```

Comment:
Why not using this? https://github.com/marketplace/actions/yq-portable-yaml-processor

Reply:
I would do that in the issue #40. I didn't want to make all the changes in one PR, otherwise it would be too confusing and the migration from version 3 to 4 still needs to be done.

Reply:
I created another pr #41 for it but it was not possible to use the action. 🙈